compose.js

```diff
@@ -8,6 +8,7 @@ import mergeWith from 'lodash/mergeWith';
 import assign from 'lodash/assign';
 import isFunction from 'lodash/isFunction';
 import isObject from 'lodash/isObject';
+import uniq from 'lodash/uniq';
```

Comment:
Not only for matter of bundle size, but I think that much more readable is to actually have `import { isFunction, isObject, uniq } from 'lodash'`.